web/www/rasa-widget/chatbot.js

```diff
@@ -0,0 +1,2 @@
+/*! For license information please see widget.js.LICENSE.txt */
```

Comment:
It is just the same file as from unpkg, but changed the font, as it was a google font.

Reply:
Does anyone whats a good practice to keep scripts like this updated? Should we just have a reminder in like 6 months to check if something changed?

Reply:
Good question, I don't know one and a quick Google doesn't turn up an obvious answer either. A reminder every six months to check all copied scripts (maybe with a growing check list of files to check?) seems like a decent solution for now.